ppl/zqd/ingest/archivepcap.go

```diff
@@ -201,8 +199,11 @@ func (p *archivePcapOp) Status() api.PcapPostStatus {
 		PcapSize:           p.pcapBytesTotal,
 		PcapReadSize:       p.pcapCounter.Bytes(),
 		DataChunksWritten:  importStats.DataChunksWritten,
-		RecordsWritten:     importStats.RecordsWritten,
 		RecordBytesWritten: importStats.RecordBytesWritten,
+		RecordsWritten:     importStats.RecordsWritten,
+		// The Brim UI uses the SnapshotCount field to determine
+		// if any data has been created in the space.
+		SnapshotCount: int(importStats.DataChunksWritten),
```

Comment:
Our place is to deprecate this Field eventually and just use DataChunksWritten, right?

Reply:
👍 I filed these two issues related to the pcap status messages:
https://github.com/brimsec/zq/issues/1980
https://github.com/brimsec/brim/issues/1359